recording_lyrics/tests/test_language_propagation.py

```diff
@@ -0,0 +1,62 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestLanguagePropagation(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.en = cls.env.ref('recording_lang.recording_language_en')
+        cls.fr = cls.env.ref('recording_lang.recording_language_fr')
+
+        cls.sound_1 = cls.env['recording'].create({
+            'name': 'Sound 1',
+            'ttype': 'sound',
+        })
+
+        cls.video_1 = cls.env['recording'].create({
+            'name': 'Video 1',
+            'ttype': 'video',
+            'sound_recording_id': cls.sound_1.id,
+        })
+
+    def _set_percentages(self, record, percentages):
+        language_vals = [(5, 0)] + [
+            (0, 0, {
+                'sequence': i,
+                'language_id': item[0].id,
+                'percentage': item[1],
+            }) for i, item in enumerate(percentages)
+        ]
+        record.write({'language_ids': language_vals})
+
+    def _assert_has_percentages(self, record, expected_percentages):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-recording-addons/pullRequest?prid=4924161)